### DISCUSSION TITLE
Super noob here, can i ask how to use this tool?

### DISCUSSION THREAD

**Discussion by ShinoIro** [FlareSolverr/FlareSolverr#393] [General]
### Environment

* **FlareSolverr version**:2.2.4
* **Last working FlareSolverr version**:First time using this
* **Operating system**: Windows 10 Pro
* **Are you using Docker**: [yes/no] no
* **FlareSolverr User-Agent (see log traces or / endpoint)**: (was it this one?)Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:101.0) Gecko/20100101 Firefox/101.0
* **Are you using a proxy or VPN?** [yes/no] yes and no (i dont know how to know that this tool is working or not, sometime when i visit site a it has cloudflare and sometimes it dont)
* **Are you using Captcha Solver:** [yes/no] no
* **If using captcha solver, which one:**
* **URL to test this issue:**

### Description

i am planing to bypass the cloudflare to use this [manga-py](https://github.com/manga-py/manga-py) to download a manga.
but since that site using cloudflare i can't do it anymore..
manga-py has support with flaresolverr but it doesnt seem to working when i paste this http://0.0.0.0:8191 to the manga-py

### Logged Error Messages

2022-05-08T21:15:55+07:00 INFO FlareSolverr v2.2.4
2022-05-08T21:15:55+07:00 INFO Testing web browser installation...
2022-05-08T21:16:05+07:00 INFO FlareSolverr User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:101.0) Gecko/20100101 Firefox/101.0
2022-05-08T21:16:07+07:00 INFO Test successful
2022-05-08T21:16:08+07:00 INFO Listening on http://0.0.0.0:8191

### Screenshots

[Place any screenshots of the issue here if needed]

**Comment by thebetauser** [comment]
https://github.com/FlareSolverr/FlareSolverr#usage

**Comment by ShinoIro** [comment]
> https://github.com/FlareSolverr/FlareSolverr#usage

still no clue tbh. the example code, should i use docker for it to be work?

**Comment by ilike2burnthing** [comment]
As far as I can tell, manga-py uses the argument `--flare-solver-url` to specify the API URL, on Windows that would just be `http://127.0.0.1:8191`.

Not sure if it's been fully implemented, so probably better to ask for clarity/instructions here - https://github.com/manga-py/manga-py/issues/413

**Reply by ShinoIro** [reply]
> As far as I can tell, manga-py uses the argument `--flare-solver-url` to specify the API URL, on Windows that would just be `http://127.0.0.1:8191`.
> 
> Not sure if it's been fully implemented, so probably better to ask for clarity/instructions here - [manga-py/manga-py#413](https://github.com/manga-py/manga-py/issues/413)

okay, i will ask this over there, thanks.. will update asap
